The function of swap causes Intel compiler to give an error. However, because this is a pure
"assert(false)" function which indicates we can safely comment it out. See the following link:

https://github.com/bcgsc/abyss/issues/230

--- abyss-2.0.3.ori/Assembly/BranchGroup.h	2019-02-01 17:55:35.908138170 -0600
+++ abyss-2.0.3/Assembly/BranchGroup.h	2019-02-01 17:55:59.978654627 -0600
@@ -209,9 +209,11 @@
 		BranchGroupStatus m_status;
 };
 
+/*
 namespace std {
 	template <>
 	inline void swap(BranchGroup&, BranchGroup&) { assert(false); }
 }
+*/
 
 #endif
